types/ember-data/index.d.ts

```diff
@@ -472,7 +473,7 @@ declare module "ember-data" {
             /**
              * Same as `deleteRecord`, but saves the record immediately.
              */
-            destroyRecord(options: {}): Promise<any>;
+            destroyRecord(options: {}): RSVP.Promise<any>;
```

Comment:
I couldn't find in the docs you linked why this change was necessary. (Or any other changes from `Promise` to `RSVP.Promise`)

Reply:
All `Promise` instances *everywhere* in Ember (including Ember Data) are `RSVP.Promise` instances. The best reference I can come up with for that is [here](https://guides.emberjs.com/v2.16.0/contributing/repositories/#toc_code-rsvp-code):

> rsvp.js - Implementation of the of Promises/A+ spec used by Ember.